.gitignore

```diff
@@ -39,3 +39,4 @@ yarn-debug.log*
 # Ignore Mac and Linux file system files
 *.swp
 .DS_Store
+.env*
```

Comment:
this file is already ignored here: # Ignore .env file containing credentials.
.env*